### ISSUE TITLE
[9.0] project/project_data date_deadline eval fails

### ISSUE THREAD

**Issue by jmalinens** [odoo/odoo#9867]
```
<field name="date_deadline" eval="'2015-%s-01' %(datetime.now().month+1)"/>
```

doesn't work if it is 2015, December. It tries to evaluate date to 2015-13-01

**Comment by smargasy** [comment]
have the same problem / can't install the module because of this bug

**Comment by ebukoz** [comment]
Having the same problem cant install the project module...

**Comment by ebukoz** [comment]
Found a work around. Locate the date_deadline field in the particular module and remove the +1

Before
<field name="date_deadline" eval="'2015-%s-01' %(datetime.now().month+1)"/>
After
<field name="date_deadline" eval="'2015-%s-01' %(datetime.now().month)"/>

**Comment by twinslady1612** [comment]
Fixed at [0ee11f2] (https://github.com/odoo/odoo/commit/0ee11f23f15ab11851baf3ca00d9c3a5947fe294)

**Comment by Yenthe666** [comment]
Indeed, this was fixed at https://github.com/odoo/odoo/commit/0ee11f23f15ab11851baf3ca00d9c3a5947fe294
